### ISSUE TITLE
Hoshi's width

### ISSUE THREAD

**Issue by mr-bat** [halilb/react-native-textinput-effects#65]
I took a look at your code of Hoshi, you're setting components width and height unrelated to the user's custom styling. And also your default width is fed from this.state which leads to undefined because you never defined the width in the state!
I partially solved the problem by a hack of not letting Hoshi set it's width itself (commenting two lines).

It's the hell I've been through lately 😅 :
<img width="78" alt="image" src="https://user-images.githubusercontent.com/16259723/31274467-d66a1008-aa9f-11e7-92ca-6e32b0950e67.png">

**Comment by mr-bat** [comment]
Also please solve the problem of rtl text for label.

**Comment by halilb** [comment]
Thank for reporting @mr-bat.

I fixed the problem and you may find the fix in version `0.4.2`.

Here is an example:

```
<Hoshi
  style={[
    styles.input,
    {
      width: 200,
    },
  ]}
  height={60}
  label={'Street'}
  maskColor={'#F9F7F6'}
  borderColor={'#7ac1ba'}
 />
```
